tket2/src/serialize/pytket/encoder.rs

```diff
@@ -586,6 +608,126 @@ impl<H: HugrView> Tk1EncoderContext<H> {
         self.commands.push(command);
     }
 
+    /// Helper to emit a `CircBox` tket1 command corresponding to a region of the Hugr.
+    ///
+    /// Returns a bool indicating whether the subcircuit was successfully emitted,
```

Comment:
nit: the return type seems a bit convoluted - is "subcircuit was not succesfully emitted" a possible error variant instead?